content/php/concepts/arrays/terms/array-reverse/array-reverse.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'array_reverse()'
+Description: 'Reverses the items in an array and returns the reversed array.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_reverse()`** function reverses the items in an array and returns that reversed array.
+
+## Syntax
+
+```pseudo
+array_reverse(array, preserve_keys)
+```
+
+- `array`: The array to reverse.
+- `preserve_keys`: Optional argument that defaults to `false` if omitted. If set to `true`, the keys of the original, non-reversed array will be preserved.
+> **Note:** If the keys used are non-numeric, they will always be preserved, regardless of whether the `preserve_keys` argument is used.
+
+## Example
+
+This example returns the reversed array `$a` using the `array_reverse()` method. In this example, the `preserve_keys` argument is not used.
+
+```php
+<?php
+  $a = array(0=>"andrew",1=>"byrne",2=>"hozier",3=>"unreal");
+  echo $backward = array_reverse($a);
+?>
+```
+
+This is the output:
+
+```shell
+Array
+(
+  [0] => unreal
+  [1] => hozier
+  [2] => byrne
+  [3] => andrew
+)
+```
+
+This example uses the `preserve_keys` argument while calling the `array_reverse()` method to return `$a`.
+
+```php
+<?php
+    $a = array(0=>"andrew",1=>"byrne",2=>"hozier",3=>"unreal");
+    echo $backward = array_reverse($a, true);
+?>
+```
+
+This is the output:
+
+```shell
+Array
+(
+  [3] => unreal
+  [2] => hozier
+  [1] => byrne
+  [0] => andrew
+)
+```
+
+## Codebyte Example
```

Comment:
The codebyte is a duplicated of the first example. I would suggest deleting it or updating the example to demonstrate some new functionality, use-case, etc

Reply:
Updated!